test/03_USM.test.js

```diff
@@ -14,8 +14,21 @@ require('chai').use(require('chai-as-promised')).should()
 
 contract('USM', (accounts) => {
   const [deployer, user1, user2, user3] = accounts
-  const [ZERO, ONE, TWO, THREE, FOUR, SIX, EIGHT, TEN, EIGHTEEN, THIRTY, HUNDRED, THOUSAND, WAD] =
-        [0, 1, 2, 3, 4, 6, 8, 10, 18, 30, 100, 1000, '1000000000000000000'].map(function (n) { return new BN(n) })
+  const [ZERO, ONE, TWO, THREE, FOUR, SIX, EIGHT, EIGHTEEN, HUNDRED, THOUSAND, WAD] = [
+    0,
+    1,
+    2,
+    3,
+    4,
+    6,
+    8,
+    18,
+    100,
+    1000,
+    '1000000000000000000',
+  ].map(function (n) {
+    return new BN(n)
+  })
```

Comment:
I dunno this doesn't read like an improvement to me...  Vertical space is valuable!

Reply:
I could disable the linter rule for it or try setting a longer target line length for prettier: https://prettier.io/docs/en/options.html#print-width (if that's what formatting it this way)